src/main/java/sam/Items.java

```diff
@@ -0,0 +1,113 @@
+package sam;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represents a task list.
+ */
+public class Items {
```

Comment:
Not sure if I like the name "Items" for this class, as it is very similar to the "Item" class. Since the header comment states that this represents a task list, maybe "TasksList" is more appropriate?